### ISSUE TITLE
Dropping item in one container but not another

### ISSUE THREAD

**Issue by sashafklein** [bevacqua/dragula#53]
Awesome tool! It's way more intuitive/light than anything else I've tried. 

I'm running into a problem, though, that I can't figure out, and seems to be throwing a warning.

I have two containers. I want to be able to copy items from the right to the left, and move them around the left, but not copy from the left to the right. 

I tried the following (in Angular, so `s` is a essentially global object):

```
var stopDrop;

s.drakeLeftRight = dragula([left, right], {
  revertOnSpill: true,
  copy: true
});

s.drakeLeft = dragula([left], {
  removeOnSpill: true
});

stopDrop = function(container) {
  if (!_.contains(container.classList, 'left')) {
    return s.drakeLeftRight.cancel();
  }
};

s.drakeLeftRight.on('drop', function(el, container, source) {
  return setTimeout((function() {
    return stopDrop(container);
  }), 0);
});
```

The hope was that the `s.drakeLeftRight.cancel()` would "cancel" the drop of the item from the left list if it wasn't dropped into the right. 

It also throws this in the console:

`Uncaught TypeError: Cannot read property 'className' of null`

But the actual behavior is currently that the item transfers (not copies) from the left to the right, disappearing from the left. I've also tried giving `cancel` an argument of the `container` into which it's being dropped (thinking, partially because of the above error, that the `revert` argument might refer to the `container` from which the item should be cancelled). 

No dice. Similarly no dice if I use `remove()`, which takes the item out from both sides. 

Any idea how to do this?

**Comment by sashafklein** [comment]
Follow-up. Is it possible to *prohibit* dropping from the left to the right, so as to preempt having to cancel the drop right after dropping it? I've tried playing around with `accepts`, but that doesn't seem to be the way to do it.

**Comment by sashafklein** [comment]
Sorry! I think I figured it out.

For future issue-searchers:

The key was, in fact, `accepts`. I set that on the `drakeLeftRight` object, so that it rejected the move if the object's `source` container was the left container, and its `target` was the right. Otherwise, it returned `true`. 

This has the side effect of deleting the object from the `left` if I try to drag it to the right, because the left panel has `removeOnSpill` set to true. But that's fine.
